src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_validators.py

```diff
@@ -68,6 +68,26 @@ def validate_keyvault(cmd, namespace):
                                           'vaults', 'Microsoft.KeyVault')
 
 
+def validate_proximity_placement_group(cmd, namespace):
+    from msrestazure.tools import parse_resource_id, is_valid_resource_id
+
+    namespace.proximity_placement_group = _get_resource_id(cmd.cli_ctx, namespace.proximity_placement_group,
+                                                           namespace.resource_group_name,
+                                                           'proximityPlacementGroups', 'Microsoft.Compute')
+
+    if not namespace.proximity_placement_group or not is_valid_resource_id(namespace.proximity_placement_group):
+        logger.debug('Proximity placement group is %s', namespace.proximity_placement_group)
```

Comment:
Is this a correct trace message when the PPD is not available?

Reply:
This is just for debugging purpose to show what was processed for the parameter. I can reword the logging message.